eahub/templates/eahub/edit_profile.html

```diff
@@ -1,81 +1,224 @@
+                  href="{% url 'account_email' %}">
+                  <i class="fa fa-pencil-alt"></i>
+                </a>
+              </div>
+            </div>
+            <div class="col">
+              <div class="form-group form-group__password">
+                <label class="control-label">Password</label>
+                <input type="text" value="●●●●●●●●●●●●●●" class="form-control" disabled>
+                <a class="btn btn-secondary btn-input-edit"
+                  href="{% url 'account_change_password' %}">
+                  <i class="fa fa-pencil-alt"></i>
+                </a>
+              </div>
+            </div>
+          </div>
+          <div class="row row-cols-2">
+            <div class="col">
+              {% include 'includes/field.html' with form=form field=form.country %}
+            </div>
+            <div class="col">
+              {% include 'includes/field.html' with form=form field=form.city_or_town %}
+            </div>
+          </div>
+          {% include 'includes/field.html' with form=form field=form.image %}
+
+          {% include 'includes/field.html' with form=form rows=7 field=form.summary placeholder="In West Philadelphia born and raised. On the playground is where I spent most of my days." %}
+
+          <div class="mb-5"
+            data-type-name="generic"
+            data-type-label="Tags"
+          >
+            <div class="vue-profile-tag-input"></div>
+          </div>
+
+          <div class="row row-cols-1 row-cols-lg-3">
+            <div class="col">
+              {% include 'includes/field.html' with form=form field=form.facebook_url %}
+            </div>
+            <div class="col">
+              {% include 'includes/field.html' with form=form field=form.linkedin_url %}
+            </div>
+            <div class="col">
+              {% include 'includes/field.html' with form=form field=form.personal_website_url placeholder="https://example.com" %}
+            </div>
+          </div>
+
+          <div class="row row-cols-1 row-cols-lg-2">
+            <div class="col">
+              {% include 'includes/field.html' with form=form rows=2 field=form.offering placeholder="Teaching python" %}
+            </div>
+            <div class="col">
+              {% include 'includes/field.html' with form=form rows=2 field=form.looking_for placeholder="Chat about wild animal suffering" %}
+            </div>
+          </div>
+
+
+          <h2>Community</h2>
+          {% include 'includes/field.html' with form=form field=form.available_as_speaker %}
+
+          {% if profile.topics_i_speak_about %}
+            {% include 'includes/field.html' with form=form rows=1 field=form.topics_i_speak_about %}
+          {% endif %}
+          <div class="mb-5"
+            data-type-name="speech_topic"
+            data-type-label="Topics I speak about"
+          >
+            <div class="vue-profile-tag-input"></div>
+          </div>
```

Comment:
![image](https://user-images.githubusercontent.com/15908208/112798637-1696a480-9065-11eb-8248-2388ea521586.png)

EAHub is missing :) @vagarwalla could you do a systematic check on your QA whether we still have all the items on each list (cause area, career interest, skills, organisational affiliation) we used to have?

Reply:
this still seems to be missing, @victor-yunenko? And did we do a check that all tags have been successully moved? Wondering because if EAHub is missing, other things might be missing, too. Or does the tags server not have the complete db?

Reply:
afaik it's called Effective Altruism Hub

but we of course should add synonyms as "eahub"

Reply:
and yes I've been testing the enums migration few times, as I needed to reset the database and rerun the migrations every week or something

so far the only issue we had is that I thought that "cause areas" are the same as "expertise areas", but it was just a silly assumption rather than a real bug